go.mod

```diff
@@ -1,179 +1,255 @@
 module github.com/Canto-Network/Canto/v7
 
-go 1.18
+go 1.21
+
+toolchain go1.21.4
 
 require (
-	github.com/armon/go-metrics v0.4.0
-	github.com/cosmos/cosmos-sdk v0.45.9
+	cosmossdk.io/api v0.7.2
+	cosmossdk.io/client/v2 v2.0.0-beta.1
+	cosmossdk.io/collections v0.4.0
+	cosmossdk.io/core v0.12.0
+	cosmossdk.io/log v1.3.0
+	cosmossdk.io/simapp v0.0.0-20231103111158-e83a20081ced
+	cosmossdk.io/store v1.0.2
+	cosmossdk.io/tools/confix v0.1.0
+	cosmossdk.io/x/circuit v0.1.0
+	cosmossdk.io/x/evidence v0.1.0
+	cosmossdk.io/x/feegrant v0.1.0
+	cosmossdk.io/x/tx v0.13.0
+	cosmossdk.io/x/upgrade v0.1.0
+	github.com/cometbft/cometbft v0.38.2
+	github.com/cosmos/cosmos-db v1.0.0
+	github.com/cosmos/cosmos-proto v1.0.0-beta.3
+	github.com/cosmos/cosmos-sdk v0.50.3
```

Comment:
sdk version v0.50.3 has a vulnerability in the vesting module, so a bump to [v0.50.4](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.4) or higher is recommended.

Reply:
bumped up cosmos v0.50.5

Reply:
The latest versions such as sdk [v0.50.6](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.6) and  cometBFT [v0.38.7](https://github.com/cometbft/cometbft/releases/tag/v0.38.7) are continuously coming out. As the PR work is getting longer, I think we can hold the bump work first and finally bump to the latest version once all the other issues are completed